templates/jail.conf.erb

```diff
@@ -15,7 +15,7 @@
 [DEFAULT]
 
 # "ignoreip" can be an IP address, a CIDR mask or a DNS host
-ignoreip = <%= node['fail2ban']['ignoreip']  %>
+ignoreip = <%= node['fail2ban']['ignoreip'].is_a?(Array) ? node['fail2ban']['ignoreip'].join(' ') : node['fail2ban']['ignoreip']  %>
```

Comment:
can we get a test case for this please? Just want to ensure it works (and we don't accidently break it in future)

Reply:
Sure 🙂